portfolio/src/main/webapp/index.html

```diff
@@ -11,34 +11,301 @@
   </head>
 
 
-  <body>
-    <section class="section">
-      <div class="container ">
-        <div class="notification is-info is-light">
-          <h1 class="title"> Bhargavi Kumar Sundaresan <i class="fas fa-hand-spock"></i></h1>
-          <p class="subtitle is-italic"> My portfolio!</p><br>
-          </div>
-        </div>
-
-        <div><br></div>
+  <body class="has-background-grey-lighter">
+	  <section class="hero is-medium is-dark is-bold">
 
-        <div class="container notification is-primary">
+		  <div class="hero-body">
+			  <div class="container">
+				  <h1 class="title"> Bhargavi Kumar Sundaresan <i class="fas fa-hand-spock"></i></h1>
+				  <h2 class="subtitle is-italic"> My portfolio! </h2>
+          <br> <br>
+          <div class="container notification">
           <article class="media">
             <div class="media-left">
               <figure class="image is-128x128">
-                <img src="/images/bks.jpg" alt="A picture of me!"> </figure>
+                <img src="/images/photo.jpg" alt="A picture of me!"> </figure>
             </div>
 
             <div class="media-content">
               <div class="content">
-                <p>
-                  <strong>A little ABOUT ME! </strong> <small> @bksundar@google.com </small> <br>
-                  Hi there! I'm Bhargavi. I was born in LA, and moved to India when I was two years old. I grew up entirely in Chennai, Tamilnadu, and did my schooling in PSBB Nungambakkam. I moved to the US in 2018 to pursue my undergraduate degree in Computer Science. Currently, I am a rising junior at the University of California, Santa cruz.
-                  </p>
+                <p class="title is-4 has-text-grey"> A little ABOUT ME!</p> 
+                <p class="title is-5 has-text-grey is-italic" > Hi there! I'm Bhargavi. I was born in LA, and moved to India when I was two years old. I grew up entirely in Chennai, Tamilnadu, and did my schooling in PSBB Nungambakkam. I moved to the US in 2018 to pursue my undergraduate degree in Computer Science. Currently, I am a rising junior at the University of California, Santa cruz.</p>
               </div>
             </div>
           </article>
         </div>
-      </section>
+
+			  </div>
+		  </div>
+	  </section>
+    
+    <section class = "section">
+      <div class="container">
+        <div class = "columns"> 
+            
+          <div class= "column">
+            <!-- This is column 1 content --> 
```

Comment:
IMO the comments can be more helpful. You can add what each column does in the comment.